### PR TITLE
Inframe and truncating driver annotation in oncoprint

### PR DESCRIPTION
**Pull request by adamabeshouse**
Inframe annotation with all data sources (hotspot, oncokb, cbioportal count, cosmic count)
Truncating annotation with oncokb only

# Checks
- [x] Runs on Heroku.
- [x] Follows [7 rules of great commit messages](http://chris.beams.io/posts/git-commit/). For most PRs a single commit should suffice, in some cases multiple topical commits can be useful. During review it is ok to see tiny commits (e.g. Fix reviewer comments), but right before the code gets merged to master or rc branch, any such commits should be squashed since they are useless to the other developers. Definitely avoid [merge commits, use rebase instead.](http://nathanleclaire.com/blog/2014/09/14/dont-be-scared-of-git-rebase/)
- [x] Follows the [Google Style Guide](https://github.com/google/styleguide).
- [x] Make sure your commit messages end with a Signed-off-by string (this line
  can be automatically added by git if you run the `git-commit` command with
  the `-s` option)
- [x] If this is a feature, the PR is to rc. If this is a bug fix, the PR is to
  hotfix.

image
index.do?cancer_study_list=nsclc_tcga_broad_2016&cancer_study_id=nsclc_tcga_broad_2016&genetic_profile_ids_PROFILE_MUTATION_EXTENDED=nsclc_tcga_broad_2016_mutations&genetic_profile_ids_PROFILE_COPY_NUMBER_ALTERATION=nsclc_tcga_broad_2016_cna&data_priority=0&case_set_id=nsclc_tcga_broad_2016_cnaseq&case_ids=&patient_case_select=sample&gene_set_choice=user-defined-list&gene_list=EGFR+MET&clinical_param_selection=null&tab_index=tab_visualize&Action=Submit&show_samples=false&

### CODE REVIEW

File: web/src/main/java/org/mskcc/cbio/portal/web/ProxyController.java [hunk 1]
```diff
@@ -98,7 +103,7 @@ public void setEnableOncokb(Boolean property) {
       Map<String, String> pathToUrl = new HashMap<>();
       
       pathToUrl.put("bitly", bitlyURL);
-      pathToUrl.put("cancerHotSpots", "http://cancerhotspots.org/api/hotspots/single/");
+      pathToUrl.put("cancerHotSpots", hotspotsURL);
```
Comment (adamabeshouse) [inline web/src/main/java/org/mskcc/cbio/portal/web/ProxyController.java]: done

File: src/main/resources/portal.properties.EXAMPLE [hunk 2]
```diff
@@ -144,6 +144,7 @@ show.oncokb=true
 
 # Enable Chang's hotspot list (true, false)
 show.hotspot=true
+hotspots.url=http://cancerhotspots.org/api/hotspots/single
```
Comment (onursumer) [inline src/main/resources/portal.properties.EXAMPLE]: Isn't this supposed to be `http://cancerhotspots.org/api/` as opposed to `http://cancerhotspots.org/api/hotspots/single`?
Seems like we are already appending `hotspots/single/` in the `ProxyController` class?

Comment (adamabeshouse) [inline src/main/resources/portal.properties.EXAMPLE]: good catch, I forgot to change it in the example

**Comment by adamabeshouse** [comment]
@jjgao @onursumer changes made and pushed

**Pull request merged**